.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 config.json
 config.hjson
 .env
+yarn.lock
```

Comment:
Should we be adding the `package-lock.json` file here too?

It also makes me think that enforcing the Node version could be useful.

Reply:
You usually want some lock file. But it is likely that these two wont be in sync so better to keep yours. 

Enforcing node version is not common and node tends to be backward compatible so I do not think we should do that

Reply:
fixed